### PR TITLE
doc: add 0.4.9 to 2.2 migration guide

### LINKED ISSUE #3399: replace the old update 0.4 -> 2.0 doc with 0.4->2.2
We now only support the export/import snapshot path.

### PR DESCRIPTION
**Pull request by xiang90**
Fix #3399

### CODE REVIEW

**Comment by philips** [comment]
This will change the indexes, etc, right? We need to document what this could break.

**Comment by xiang90** [comment]
@philips Updated.

File: Documentation/04_to_2_snapshot_migration.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+## Snapshot Migration
+
+You can migrate a snapshot of your data from a v0.4.9+ cluster into a new etcd 2.2 cluster using a snapshot migration. After snapshot migration, the etcd indexes of your data will change.
```
Comment (philips) [inline Documentation/04_to_2_snapshot_migration.md]: Something like this: Many etcd applications rely on these indexes to behave correctly so this operation should only be done while all applications are stopped.

**Comment by xiang90** [comment]
@philips All resolved. I am going to merge this.

**Pull request merged**